protocols/grpc/hyperstart.proto

```diff
@@ -110,20 +116,55 @@ message CreateSandboxRequest {
 message DestroySandboxRequest {
 }
 
-message IpAddress {
-	string address = 1;
-	string mask = 2;
+enum IPFamily {
+	v4 = 0;
+	v6 = 1;
+}
+
+message IPAddress {
+	IPFamily family = 1;
+	string address = 2;
+	string mask = 3;
+}
+
+message Interface {
+	string device = 1;
+	string name = 2;
+	repeated IPAddress ipAddresses = 3;
+	uint64 mtu = 4;
+	string hwAddr = 5;
+}
+
+message Route {
+	string dest = 1;
+	string gateway = 2;
+	string device = 3;
+}
+
+enum UpdateType {
+	IP = 0;
```

Comment:
one `IP` field can't distinguish "add ip" or "remove" ip, for interfaces could have many ip addresses.

Reply:
@WeiZhang555 Fixed.